simulation/src/test/resources/live_reloading[CONCURRENCY_LIMITER].txt

```diff
@@ -0,0 +1 @@
+success=58.3%	client_mean=PT0.7228S      	server_cpu=PT4M49.12S     	received=400/400	codes={200=233, 500=167}
```

Comment:
Seems like github hides the pngs by default - gotta click the little "Display the rich diff" thing:

![image](https://user-images.githubusercontent.com/3473798/74658220-b4911d80-5189-11ea-839d-e45cf6748302.png)